### DISCUSSION TITLE
How to trigger reauthentication?

### DISCUSSION THREAD

**Discussion by CongMs** [community/community#42266] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I logged in the extension with an old Github account before, which doesn't have access to Copilot. Now I would like to switch to use a new account, but there is no option to re-login, and it keeps prompting "Invalid copilot token: missing token: 403." Is there any way to trigger the authentication again? I've tried uninstalling and reinstalling the extension but it didn't work.

**Comment by iamcjwilliams** [comment]
@CongMs Which IDE are you using?

**Comment by CongMs** [comment]
I tried both VS code and Visual Studio. They behave the same.

**Comment by kimadeline** [comment, marked as answer]
Hi @CongMs, there should be an "Accounts" icon in your VS Code side bar where you can sign out of your GitHub account, and sign in again:

![image](https://user-images.githubusercontent.com/51720070/209412794-aaba6baf-fb55-407e-beab-020355bb0589.png)

**Comment by CongMs** [comment]
That works for me. Thanks!
